tools/mcmanifest.js

```diff
@@ -1946,8 +1946,14 @@ export class Tool extends TOOL {
 
 				flows.forEach((node, i) => {
 					let manifest;
-					if (node.moddable_manifest)
-						manifest = {...node.moddable_manifest};
+					if (node.moddable_manifest){
```

Comment:
My understanding is that this change was made to allow `moddable_manifest` to be a String containing JSON. As implemented, if there is a syntax error in the JSON, it will be ignored instead of reported. That will be confusing for developers. I suggest reworking the change as:

```js
	if (node.moddable_manifest) {
		if ("string" === typeof node.moddable_manifest)
			manifest = JSON.parse(node.moddable_manifest);
		else
			manifest = {...node.moddable_manifest};
	}
```

Reply:
It would be better if you make a suggestion.
thank you.